[libgwenhywfar] Build failue

Package: libgwenhywfar[core,qt5]:arm64-osx -> 5.6.0#3

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: a5d91f7d2 2023-05-01 (17 hours ago)

**To Reproduce**

`vcpkg x-set-installed libgwenhywfar[core,qt5]`

**Failure logs**

```
-- Using cached gwenhywfar-5.6.0.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/gwenhywfar-5.6.0.tar.gz
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libgwenhywfar/src/5.6.0-7b89005ea8.clean
-- Found external ninja('1.11.1').
-- Getting CMake variables for arm64-osx-dbg
-- Getting CMake variables for arm64-osx-rel
-- Configuring arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /bin/bash -c "V=1 ./../src/5.6.0-7b89005ea8.clean/configure  \"--disable-silent-rules\" \"--disable-binreloc\" \"--with-guis=qt5\" \"--with-libgpg-error-prefix=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/libgpg-error\" \"--with-libgcrypt-prefix=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/libgcrypt\" \"--with-qt5-qmake=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/qt5/bin/qmake\" \"--with-qt5-moc=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/qt5/bin/moc\" \"--with-qt5-uic=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/qt5/bin/uic\" \"LDFLAGS=-framework CoreFoundation -framework Security\" \"--disable-silent-rules\" \"--verbose\" \"--disable-shared\" \"--enable-static\" \"--prefix=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug\" \"--bindir=\\${prefix}/../tools/libgwenhywfar/debug/bin\" \"--sbindir=\\${prefix}/../tools/libgwenhywfar/debug/sbin\" \"--libdir=\\${prefix}/lib\" \"--includedir=\\${prefix}/../include\" \"--datarootdir=\\${prefix}/share/libgwenhywfar\""
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libgwenhywfar/arm64-osx-dbg
    Error code: 1
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libgwenhywfar/config-arm64-osx-dbg-config.log
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libgwenhywfar/config-arm64-osx-dbg-out.log
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libgwenhywfar/config-arm64-osx-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_make.cmake:833 (vcpkg_execute_required_process)
  ports/libgwenhywfar/portfile.cmake:36 (vcpkg_configure_make)
  scripts/ports.cmake:147 (include)



```

<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libgwenhywfar/config-arm64-osx-dbg-out.log</summary>

```
checking build system type... arm-apple-darwin22.4.0
checking host system type... arm-apple-darwin22.4.0
checking for a BSD-compatible install... /opt/homebrew/bin/ginstall -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /opt/homebrew/bin/gmkdir -p
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking host system type... posix
checking for gcc... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g accepts -g... yes
checking for /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g option to accept ISO C89... none needed
checking whether /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g understands -c and -o together... yes
checking for style of include used by make... GNU
checking dependency style of /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g... gcc3
checking whether we are using the GNU C++ compiler... yes
checking whether /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ accepts -g... yes
checking dependency style of /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++... gcc3
checking how to run the C preprocessor... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g -E
checking for gcc... gcc
checking whether we are using the GNU Objective C compiler... yes
checking whether gcc accepts -g... yes
checking dependency style of gcc... gcc3
checking how to print strings... printf
checking for a sed that does not truncate output... /usr/bin/sed
checking for grep that handles long lines and -e... /opt/homebrew/bin/ggrep
checking for egrep... /opt/homebrew/bin/ggrep -E
checking for fgrep... /opt/homebrew/bin/ggrep -F
checking for ld used by /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
checking if the linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) is GNU ld... no
checking for BSD- or MS-compatible name lister (nm)... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/nm
checking the name lister (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/nm) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 786432
checking how to convert arm-apple-darwin22.4.0 file names to arm-apple-darwin22.4.0 format... func_convert_file_noop
checking how to convert arm-apple-darwin22.4.0 file names to toolchain format... func_convert_file_noop
checking for /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld option to reload object files... -r
checking for objdump... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for archiver @FILE support... no
checking for strip... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/strip
checking for ranlib... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib
checking command to parse /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/nm output from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for mt... no
checking if : is a manifest tool... no
checking for dsymutil... dsymutil
checking for nmedit... nmedit
checking for lipo... lipo
checking for otool... otool
checking for otool64... no
checking for -single_module linker flag... yes
checking for -exported_symbols_list linker flag... yes
checking for -force_load linker flag... yes
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g supports -fno-rtti -fno-exceptions... yes
checking for /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g option to produce PIC... -fno-common -DPIC
checking if /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g PIC flag -fno-common -DPIC works... yes
checking if /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g static flag -static works... no
checking if /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g supports -c -o file.o... yes
checking if /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g supports -c -o file.o... (cached) yes
checking whether the /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin22.4.0 dyld
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... no
checking whether to build static libraries... yes
checking how to run the C++ preprocessor... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -E
checking for ld used by /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
checking if the linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) is GNU ld... no
checking whether the /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries... yes
checking for /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ option to produce PIC... -fno-common -DPIC
checking if /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ PIC flag -fno-common -DPIC works... yes
checking if /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ static flag -static works... no
checking if /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ supports -c -o file.o... yes
checking if /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ supports -c -o file.o... (cached) yes
checking whether the /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin22.4.0 dyld
checking how to hardcode library paths into programs... immediate
checking for windres... no
checking for windres... no
checking for gawk... (cached) awk
checking for dot... YES
checking for sed... /usr/bin/sed
checking for astyle... echo
checking for dlopen in -ldl... yes
checking whether /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -g is Clang... yes
checking whether Clang needs flag to prevent "argument unused" warning when linking with -pthread... no
checking for joinable pthread attribute... PTHREAD_CREATE_JOINABLE
checking whether more special flags are required for pthreads... no
checking for PTHREAD_PRIO_INHERIT... yes
checking for ld used by GCC... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
checking if the linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) is GNU ld... no
checking for shared library run path origin... done
checking for iconv... yes
checking for working iconv... yes
checking how to link with libiconv... -liconv
checking for iconv declaration... 
         extern size_t iconv (iconv_t cd, char * *inbuf, size_t *inbytesleft, char * *outbuf, size_t *outbytesleft);
checking for dirent.h that defines DIR... yes
checking for library containing opendir... none required
checking for ANSI C header files... (cached) yes
checking arpa/inet.h usability... yes
checking arpa/inet.h presence... yes
checking for arpa/inet.h... yes
checking fcntl.h usability... yes
checking fcntl.h presence... yes
checking for fcntl.h... yes
checking netdb.h usability... yes
checking netdb.h presence... yes
checking for netdb.h... yes
checking netinet/in.h usability... yes
checking netinet/in.h presence... yes
checking for netinet/in.h... yes
checking for stdlib.h... (cached) yes
checking for string.h... (cached) yes
checking for strings.h... (cached) yes
checking sys/socket.h usability... yes
checking sys/socket.h presence... yes
checking for sys/socket.h... yes
checking sys/time.h usability... yes
checking sys/time.h presence... yes
checking for sys/time.h... yes
checking syslog.h usability... yes
checking syslog.h presence... yes
checking for syslog.h... yes
checking for unistd.h... (cached) yes
checking for unistd.h... (cached) yes
checking for dlfcn.h... (cached) yes
checking signal.h usability... yes
checking signal.h presence... yes
checking for signal.h... yes
checking termios.h usability... yes
checking termios.h presence... yes
checking for termios.h... yes
checking langinfo.h usability... yes
checking langinfo.h presence... yes
checking for langinfo.h... yes
checking localcharset.h usability... yes
checking localcharset.h presence... yes
checking for localcharset.h... yes
checking iconv.h usability... yes
checking iconv.h presence... yes
checking for iconv.h... yes
checking time.h usability... yes
checking time.h presence... yes
checking for time.h... yes
checking locale.h usability... yes
checking locale.h presence... yes
checking for locale.h... yes
checking for syslog.h... (cached) yes
checking for sys/stat.h... (cached) yes
checking for sys/types.h... (cached) yes
checking sys/wait.h usability... yes
checking sys/wait.h presence... yes
checking for sys/wait.h... yes
checking regex.h usability... yes
checking regex.h presence... yes
checking for regex.h... yes
checking endian.h usability... no
checking endian.h presence... no
checking for endian.h... no
checking whether sigprocmask is declared... yes
checking for an ANSI C-conforming const... yes
checking whether time.h and sys/time.h may both be included... yes
checking whether struct tm is in sys/time.h or time.h... time.h
checking whether closedir returns void... no
checking for sys/time.h... (cached) yes
checking for unistd.h... (cached) yes
checking for alarm... yes
checking for working mktime... no
checking sys/select.h usability... yes
checking sys/select.h presence... yes
checking for sys/select.h... yes
checking for sys/socket.h... (cached) yes
checking types of arguments for select... int,fd_set *,struct timeval *
checking for gethostbyaddr... yes
checking for gethostbyname... yes
checking for inet_ntoa... yes
checking for isascii... yes
checking for getpid... yes
checking for memmove... yes
checking for memset... yes
checking for select... yes
checking for socket... yes
checking for strcasecmp... yes
checking for strchr... yes
checking for strdup... yes
checking for strerror... yes
checking for snprintf... yes
checking for symlink... yes
checking for setlocale... yes
checking for srandom... yes
checking for random... yes
checking whether binary relocation support should be enabled... no
checking if gwen is a subproject... no
checking whether system-wide trusted certificates should be set... yes
checking network connectivity can be assumed available during 'make check'... yes
checking whether to enable test code... no
checking whether local installation mode is wanted... no
checking plugin search dir... ${libdir}/gwenhywfar/plugins/79
checking data search dir... ${datadir}/gwenhywfar
checking system data search dir... ${datadir}
checking locale search dir... $(localedir)
checking cfg search dir... $(sysconfdir)
checking if compiling with variadic macros... yes
checking if profiling is to be supported... no
checking for dpkg-architecture... no
checking for multiarch paths... not found
checking for libgcrypt-config... /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/libgcrypt/bin/libgcrypt-config
checking for LIBGCRYPT - version >= 1.2.0... yes (1.10.1-unknown)
checking LIBGCRYPT API version... okay
checking for gpg-error-config... /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/libgpg-error/bin/gpg-error-config
checking for GPG Error - version >= 0.0... yes (1.46-unknown)
checking pkg-config is at least version 0.9.0... yes
checking for LIBGNUTLS... yes
checking for OPENSSL... yes
checking for preselected distribution name... no, will have to determine it
checking whether this is a Debian derivate... no
checking whether this is an RPM distribution... no (no RPM installed)
checking for xgettext... xgettext
checking for msgfmt... /opt/homebrew/bin/msgfmt
checking for locale.h... (cached) yes
checking libintl.h usability... yes
checking libintl.h presence... yes
checking for libintl.h... yes
checking for library containing gettext... no
checking for iconv.h... (cached) yes
checking for iconv... no
checking for iconv in -liconv... yes
checking libXML2-depending code... no
checking for X... no
checking for Qt... yes:
    QT_CXXFLAGS= 
    QT_DIR=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib
    QT_LIBS= 
    QT_UIC=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/qt5/bin/uic
    QT_MOC=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/qt5/bin/moc
    QT_LRELEASE=
    QT_LUPDATE=
checking correct functioning of Qt installation... failure
```
</details>
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libgwenhywfar/config-arm64-osx-dbg-err.log</summary>

```
rm: conftest.dSYM: is a directory
rm: conftest.dSYM: is a directory
rm: conftest.dSYM: is a directory
rm: conftest.dSYM: is a directory
configure: WARNING:
***
*** The config script /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/libgcrypt/bin/libgcrypt-config was
*** built for aarch64-apple-darwin22.4.0 and thus may not match the
*** used host arm-apple-darwin22.4.0.
*** You may want to use the configure option --with-libgcrypt-prefix
*** to specify a matching config script or use $SYSROOT.
***
configure: WARNING:
***
*** The config script /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/libgpg-error/bin/gpg-error-config was
*** built for aarch64-apple-darwin22.4.0 and thus may not match the
*** used host arm-apple-darwin22.4.0.
*** You may want to use the configure option --with-libgpg-error-prefix
*** to specify a matching config script or use $SYSROOT.
***
Project WARNING: Qt has only been tested with version 12 of the platform SDK, you're using 13.
Project WARNING: This is an unsupported configuration. You may experience build issues, and by using
Project WARNING: the 13.3 SDK you are opting in to new features that Qt has not been prepared for.
Project WARNING: Please downgrade the SDK you use to build your app to version 12, or configure
Project WARNING: with CONFIG+=sdk_no_version_check when running qmake to silence this warning.
Cannot find file: tmp.WBwf1ID8.
make: *** [/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T/tmp.14Jq7sGX] Error 2
Cannot find file: tmp.WBwf1ID8.
make: *** [/var/folders/2t/jzczznpx033c6tbrdt24ldhw0000gn/T/tmp.14Jq7sGX] Error 2
configure: error: Failed to find matching components of a complete
                  Qt installation. Try using more options,
                  see ./configure --help.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "gz-tools2",
      "default-features": false,
      "features": []
    }
  ],
  "$overrides": [
    {
      "name": "capstone",
      "version": "4.0.1-120373dc"
    }
  ],
  "$builtin-baseline": "338f99eed499b6df1628829e40695b396a70a1f7"
}

```
</details>

--- a/ports/libgwenhywfar/control-openssl.diff
+++ b/ports/libgwenhywfar/control-openssl.diff
@@ -1,0 +1,20 @@
+diff --git a/configure.ac b/configure.ac
+index 4e43f5c..7ce9511 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -709,7 +709,14 @@ PKG_CHECK_MODULES([LIBGNUTLS], [gnutls >= 2.9.8])
+ #
+ # check for OpenSSL
+ #
+-PKG_CHECK_MODULES([OPENSSL], [openssl], [ssl_available=yes], [ssl_available=no])
++AC_ARG_WITH(openssl, [  --with-openssl=yes      enable OpenSSL in gct-tool],
++  [with_openssl="$withval"],
++  [with_openssl="no"])
++if test "x$with_openssl" = "xyes"; then
++  PKG_CHECK_MODULES([OPENSSL], [openssl], [ssl_available=yes], [AC_MSG_ERROR([unable to find OpenSSL (pkg-config)])])
++else
++  ssl_available=no
++fi
+ if test "$ssl_available" = "yes"; then
+   AC_DEFINE_UNQUOTED(HAVE_OPENSSL, 1, [Defined if OpenSSL is available])
+ fi

--- a/ports/libgwenhywfar/pkgconfig.diff
+++ b/ports/libgwenhywfar/pkgconfig.diff
@@ -1,0 +1,11 @@
+diff --git a/gwenhywfar.pc.in b/gwenhywfar.pc.in
+index d4c61d8..72a4ef4 100644
+--- a/gwenhywfar.pc.in
++++ b/gwenhywfar.pc.in
+@@ -31,3 +31,6 @@ Version: @VERSION@
+ Libs: @gwenhywfar_ldflags@ @gwenhywfar_libs@
+ Cflags: -I@gwenhywfar_headerdir@
+ 
++Libs.private: @all_libraries@
++@WITH_LIBXML2_CODE_FALSE@Requires.private: libgcrypt, gnutls
++@WITH_LIBXML2_CODE_TRUE@Requires.private: libgcrypt, gnutls, libxml-2.0

--- a/ports/libgwenhywfar/portfile.cmake
+++ b/ports/libgwenhywfar/portfile.cmake
@@ -1,13 +1,15 @@
 vcpkg_download_distfile(ARCHIVE
-    URLS "https://www.aquamaniac.de/rdm/attachments/download/364/gwenhywfar-${VERSION}.tar.gz"
+    URLS "https://www.aquamaniac.de/rdm/attachments/download/529/gwenhywfar-5.12.0.tar.gz"
     FILENAME "gwenhywfar-${VERSION}.tar.gz"
-    SHA512 9875d677f49fc0a46f371fd1954d15d99c7d5994e90b16f1be7a5b8a1cbcd74ae9733e4541afd6d8251a2ba1a0a37c28e0f248952b7c917313fbf5b38b1d8d11
+    SHA512 0075eb626f0022ecd4ffdd59de7f0817d2def685e1d2cfbca9a32faa4b8d4d213bea631f24c5385da0b8c7743fd6d1887a46f08afa371195d911409ec7655791
 )
 
 vcpkg_extract_source_archive(
     SOURCE_PATH
     ARCHIVE "${ARCHIVE}"
     SOURCE_BASE "${VERSION}"
+    PATCHES
+        static-link-order.diff
 )
 
 vcpkg_list(SET options)
@@ -24,32 +26,31 @@ endif()
 list(JOIN FEATURES_GUI " " GUIS)
 vcpkg_list(APPEND options "--with-guis=${GUIS}")
 
-if(VCPKG_TARGET_IS_OSX)
-    vcpkg_list(APPEND options "LDFLAGS=\$LDFLAGS -framework CoreFoundation -framework Security")
-endif()
-
-vcpkg_configure_make(
+set(ENV{ACLOCAL} "aclocal -I \"${CURRENT_INSTALLED_DIR}/share/libgpg-error/aclocal/\"")
+vcpkg_make_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
         --disable-binreloc
+        --disable-network-checks
+        --disable-nls
         ${options}
     OPTIONS_RELEASE
         "--with-qt5-qmake=${CURRENT_HOST_INSTALLED_DIR}/tools/qt5/bin/qmake"
         "--with-qt5-moc=${CURRENT_HOST_INSTALLED_DIR}/tools/qt5/bin/moc"
         "--with-qt5-uic=${CURRENT_HOST_INSTALLED_DIR}/tools/qt5/bin/uic"
-        "GPG_ERROR_CONFIG=${CURRENT_INSTALLED_DIR}/tools/libgpg-error/bin/gpgrt-config"
-        "gpg_error_config_args=gpg-error"
+        "GPG_ERROR_CONFIG=${CURRENT_INSTALLED_DIR}/tools/libgpg-error/bin/gpgrt-config gpg-error"
+        "GPGRT_CONFIG=${CURRENT_INSTALLED_DIR}/tools/libgpg-error/bin/gpgrt-config"
         "LIBGCRYPT_CONFIG=${CURRENT_INSTALLED_DIR}/tools/libgcrypt/bin/libgcrypt-config"
     OPTIONS_DEBUG
         "--with-qt5-qmake=${CURRENT_HOST_INSTALLED_DIR}/tools/qt5/debug/bin/qmake"
         "--with-qt5-moc=${CURRENT_HOST_INSTALLED_DIR}/tools/qt5/debug/bin/moc"
         "--with-qt5-uic=${CURRENT_HOST_INSTALLED_DIR}/tools/qt5/debug/bin/uic"
-        "GPG_ERROR_CONFIG=${CURRENT_INSTALLED_DIR}/tools/libgpg-error/debug/bin/gpgrt-config"
-        "gpg_error_config_args=gpg-error"
+        "GPG_ERROR_CONFIG=${CURRENT_INSTALLED_DIR}/tools/libgpg-error/debug/bin/gpgrt-config gpg-error"
+        "GPGRT_CONFIG=${CURRENT_INSTALLED_DIR}/tools/libgpg-error/debug/bin/gpgrt-config"
         "LIBGCRYPT_CONFIG=${CURRENT_INSTALLED_DIR}/tools/libgcrypt/debug/bin/libgcrypt-config"
 )
 
-vcpkg_install_make()
+vcpkg_make_install()
 vcpkg_copy_pdbs()
 vcpkg_fixup_pkgconfig()
 
@@ -59,15 +60,9 @@ foreach(GUI IN LISTS FEATURES_GUI)
 endforeach()
 vcpkg_cmake_config_fixup(PACKAGE_NAME gwenhywfar CONFIG_PATH lib/cmake/gwenhywfar-${MAJOR_MINOR})
 
-if ("tools" IN_LIST FEATURES)
-    vcpkg_copy_tools(SEARCH_DIR "${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin" TOOL_NAMES gct-tool gsa mklistdoc typemaker typemaker2 xmlmerge AUTO_CLEAN)
-endif()
-
-# the `dir` variable is not used in the script
-vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/libgwenhywfar/bin/gwenhywfar-config" "dir=\"${CURRENT_INSTALLED_DIR}\"" "")
-if(NOT VCPKG_BUILD_TYPE)
-    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/libgwenhywfar/debug/bin/gwenhywfar-config" "dir=\"${CURRENT_INSTALLED_DIR}/debug\"" "")
-endif()
+vcpkg_copy_tool_dependencies("${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/tools/${PORT}/debug/bin")
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
+
 vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/COPYING")

--- a/ports/libgwenhywfar/portfile.cmake
+++ b/ports/libgwenhywfar/portfile.cmake
@@ -75,7 +75,8 @@ endforeach()
 vcpkg_cmake_config_fixup(PACKAGE_NAME gwenhywfar CONFIG_PATH lib/cmake/gwenhywfar-${MAJOR_MINOR})
 
 vcpkg_copy_tool_dependencies("${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin")
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/tools/${PORT}/debug/bin")
+vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin/gwenhywfar-config" [[dir="[^"]*"]] [[dir=""]] REGEX) # unused abs path
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/tools/${PORT}/debug")
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 

--- a/ports/libgwenhywfar/portfile.cmake
+++ b/ports/libgwenhywfar/portfile.cmake
@@ -4,12 +4,19 @@ vcpkg_download_distfile(ARCHIVE
     SHA512 0075eb626f0022ecd4ffdd59de7f0817d2def685e1d2cfbca9a32faa4b8d4d213bea631f24c5385da0b8c7743fd6d1887a46f08afa371195d911409ec7655791
 )
 
+vcpkg_download_distfile(osx_patch
+    URLS "https://www.aquamaniac.de/rdm/projects/gwenhywfar/repository/revisions/55d4b7b526df30e4003c92e2f504f480c01021f0/diff?format=diff"
+    FILENAME "gwenhywfar-5.12.0-55d4b7b.diff"
+    SHA512 87fa9ff3e9027c5a6839f800990b420a824efbd115ed67eeaef3c909b14c59c0b9bae41c539d400166862c0353ad730313ee4f9366928c333883d41429912731
+)
+
 vcpkg_extract_source_archive(
     SOURCE_PATH
     ARCHIVE "${ARCHIVE}"
     SOURCE_BASE "${VERSION}"
     PATCHES
         static-link-order.diff
+        ${osx_patch}
 )
 
 vcpkg_list(SET options)

--- a/ports/libgwenhywfar/portfile.cmake
+++ b/ports/libgwenhywfar/portfile.cmake
@@ -15,6 +15,7 @@ vcpkg_extract_source_archive(
     ARCHIVE "${ARCHIVE}"
     SOURCE_BASE "${VERSION}"
     PATCHES
+        control-openssl.diff
         static-link-order.diff
         ${osx_patch}
 )
@@ -33,9 +34,15 @@ endif()
 list(JOIN FEATURES_GUI " " GUIS)
 vcpkg_list(APPEND options "--with-guis=${GUIS}")
 
-set(ENV{ACLOCAL} "aclocal -I \"${CURRENT_INSTALLED_DIR}/share/libgpg-error/aclocal/\"")
+if ("openssl" IN_LIST FEATURES)
+    vcpkg_list(APPEND options "--with-openssl=yes")
+endif()
+
+set(ENV{ACLOCAL} "aclocal -I \"${CURRENT_INSTALLED_DIR}/share/libgpg-error/aclocal/\" -I \"${CURRENT_INSTALLED_DIR}/share/libgcrypt/aclocal/\" -I \"${CURRENT_HOST_INSTALLED_DIR}/share/gettext/aclocal/\"")
+set(ENV{AUTOPOINT} true)
 vcpkg_make_configure(
     SOURCE_PATH "${SOURCE_PATH}"
+    AUTORECONF
     OPTIONS
         --disable-binreloc
         --disable-network-checks

--- a/ports/libgwenhywfar/portfile.cmake
+++ b/ports/libgwenhywfar/portfile.cmake
@@ -16,6 +16,7 @@ vcpkg_extract_source_archive(
     SOURCE_BASE "${VERSION}"
     PATCHES
         control-openssl.diff
+        pkgconfig.diff
         static-link-order.diff
         ${osx_patch}
 )
@@ -68,11 +69,15 @@ vcpkg_make_install()
 vcpkg_copy_pdbs()
 vcpkg_fixup_pkgconfig()
 
-string(REGEX MATCH "^([0-9]*[.][0-9]*)" MAJOR_MINOR "${VERSION}")
-foreach(GUI IN LISTS FEATURES_GUI)
-    vcpkg_cmake_config_fixup(PACKAGE_NAME gwengui-${GUI} CONFIG_PATH lib/cmake/gwengui-${GUI}-${MAJOR_MINOR} DO_NOT_DELETE_PARENT_CONFIG_PATH)
-endforeach()
-vcpkg_cmake_config_fixup(PACKAGE_NAME gwenhywfar CONFIG_PATH lib/cmake/gwenhywfar-${MAJOR_MINOR})
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
+    string(REGEX MATCH "^([0-9]*[.][0-9]*)" MAJOR_MINOR "${VERSION}")
+    foreach(GUI IN LISTS FEATURES_GUI)
+        vcpkg_cmake_config_fixup(PACKAGE_NAME gwengui-${GUI} CONFIG_PATH lib/cmake/gwengui-${GUI}-${MAJOR_MINOR} DO_NOT_DELETE_PARENT_CONFIG_PATH)
+    endforeach()
+    vcpkg_cmake_config_fixup(PACKAGE_NAME gwenhywfar CONFIG_PATH lib/cmake/gwenhywfar-${MAJOR_MINOR})
+else()
+    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib/cmake" "${CURRENT_PACKAGES_DIR}/debug/lib/cmake")
+endif()
 
 vcpkg_copy_tool_dependencies("${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin")
 vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin/gwenhywfar-config" [[dir="[^"]*"]] [[dir=""]] REGEX) # unused abs path

--- a/ports/libgwenhywfar/static-link-order.diff
+++ b/ports/libgwenhywfar/static-link-order.diff
@@ -1,0 +1,17 @@
+diff --git a/tools/gwenbuild/Makefile.am b/tools/gwenbuild/Makefile.am
+index 14aa0e2..eb60988 100644
+--- a/tools/gwenbuild/Makefile.am
++++ b/tools/gwenbuild/Makefile.am
+@@ -28,11 +28,11 @@ noinst_HEADERS=\
+ 
+ 
+ gwbuild_LDADD = \
+-  $(top_builddir)/src/$(gwenhywfar_internal_libname) \
+   types/libgwb_types.la \
+   parser/libgwb_parser.la \
+   tools/libgwb_tools.la \
+   builders/libgwb_builders.la \
+   buildctx/libgwb_build.la \
++  $(top_builddir)/src/$(gwenhywfar_internal_libname) \
+   $(PROG_LIBRARIES)
+ 

--- a/ports/libgwenhywfar/vcpkg.json
+++ b/ports/libgwenhywfar/vcpkg.json
@@ -39,15 +39,14 @@
     },
     "openssl": {
       "description": "OpenSSL support in gct-tool",
+      "supports": "!arm64",
       "dependencies": [
-        {
-          "name": "openssl",
-          "platform": "!arm64"
-        }
+        "openssl"
       ]
     },
     "qt5": {
       "description": "Qt bindings",
+      "supports": "!(osx & static)",
       "dependencies": [
         {
           "name": "qt5-base",

--- a/ports/libgwenhywfar/vcpkg.json
+++ b/ports/libgwenhywfar/vcpkg.json
@@ -5,9 +5,12 @@
   "homepage": "https://www.aquamaniac.de/rdm/",
   "supports": "!windows | mingw",
   "dependencies": [
+    {
+      "name": "gettext",
+      "host": true
+    },
     "libgcrypt",
     "libgnutls",
-    "openssl",
     {
       "name": "vcpkg-cmake-config",
       "host": true
@@ -31,6 +34,15 @@
         {
           "name": "libxml2",
           "default-features": false
+        }
+      ]
+    },
+    "openssl": {
+      "description": "OpenSSL support in gct-tool",
+      "dependencies": [
+        {
+          "name": "openssl",
+          "platform": "!arm64"
         }
       ]
     },

--- a/ports/libgwenhywfar/vcpkg.json
+++ b/ports/libgwenhywfar/vcpkg.json
@@ -1,20 +1,19 @@
 {
   "name": "libgwenhywfar",
-  "version-semver": "5.6.0",
-  "port-version": 5,
+  "version-semver": "5.12.0",
   "description": "A helper library for networking and security applications and libraries",
   "homepage": "https://www.aquamaniac.de/rdm/",
-  "supports": "!windows & !(osx & arm64)",
+  "supports": "!windows | mingw",
   "dependencies": [
     "libgcrypt",
-    {
-      "name": "libgnutls",
-      "features": [
-        "openssl"
-      ]
-    },
+    "libgnutls",
+    "openssl",
     {
       "name": "vcpkg-cmake-config",
+      "host": true
+    },
+    {
+      "name": "vcpkg-make",
       "host": true
     }
   ],
@@ -24,7 +23,7 @@
   ],
   "features": {
     "cpp": {
-      "description": "C++ bindings"
+      "description": "A C++ binding for the C module GWEN_GUI"
     },
     "libxml2": {
       "description": "Enables libXML2-depending functionality",
@@ -43,9 +42,6 @@
           "default-features": false
         }
       ]
-    },
-    "tools": {
-      "description": "Some helper tools provided by Gwenhywfar and useful for applications using it"
     }
   }
 }

--- a/scripts/test_ports/vcpkg-ci-libgwenhywfar/portfile.cmake
+++ b/scripts/test_ports/vcpkg-ci-libgwenhywfar/portfile.cmake
@@ -1,0 +1,10 @@
+set(VCPKG_POLICY_EMPTY_PACKAGE enabled)
+
+vcpkg_find_acquire_program(PKGCONFIG)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${CURRENT_PORT_DIR}/project"
+    OPTIONS
+        "-DPKG_CONFIG_EXECUTABLE=${PKGCONFIG}"
+)
+vcpkg_cmake_build()

--- a/scripts/test_ports/vcpkg-ci-libgwenhywfar/project/CMakeLists.txt
+++ b/scripts/test_ports/vcpkg-ci-libgwenhywfar/project/CMakeLists.txt
@@ -1,0 +1,11 @@
+cmake_minimum_required(VERSION 3.10)
+project(libgwenhywfar-test C)
+
+find_package(PkgConfig REQUIRED)
+
+block()
+    pkg_check_modules(gwenhywfar IMPORTED_TARGET REQUIRED gwenhywfar)
+
+    add_executable(libgwenhywfar_pkgconfig main.c)
+    target_link_libraries(libgwenhywfar_pkgconfig PkgConfig::gwenhywfar)
+endblock()

--- a/scripts/test_ports/vcpkg-ci-libgwenhywfar/project/main.c
+++ b/scripts/test_ports/vcpkg-ci-libgwenhywfar/project/main.c
@@ -1,0 +1,8 @@
+#include <gwenhywfar/gwenhywfar.h>  /* based on gwenhywfar.pc */
+
+int main()
+{
+    int result = GWEN_Init();
+    GWEN_Fini();
+    return result;
+}

--- a/scripts/test_ports/vcpkg-ci-libgwenhywfar/vcpkg.json
+++ b/scripts/test_ports/vcpkg-ci-libgwenhywfar/vcpkg.json
@@ -1,0 +1,46 @@
+{
+  "name": "vcpkg-ci-libgwenhywfar",
+  "version-string": "ci",
+  "description": "Validates libgwenhywfar",
+  "dependencies": [
+    {
+      "name": "libgwenhywfar",
+      "default-features": false
+    },
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    }
+  ],
+  "default-features": [
+    {
+      "name": "openssl",
+      "platform": "!arm64"
+    },
+    "qt5"
+  ],
+  "features": {
+    "openssl": {
+      "description": "Use openssl",
+      "dependencies": [
+        {
+          "name": "libgwenhywfar",
+          "features": [
+            "openssl"
+          ]
+        }
+      ]
+    },
+    "qt5": {
+      "description": "Install qt5 binding",
+      "dependencies": [
+        {
+          "name": "libgwenhywfar",
+          "features": [
+            "qt5"
+          ]
+        }
+      ]
+    }
+  }
+}

--- a/scripts/test_ports/vcpkg-ci-libgwenhywfar/vcpkg.json
+++ b/scripts/test_ports/vcpkg-ci-libgwenhywfar/vcpkg.json
@@ -17,7 +17,10 @@
       "name": "openssl",
       "platform": "!arm64"
     },
-    "qt5"
+    {
+      "name": "qt5",
+      "platform": "!(osx & static)"
+    }
   ],
   "features": {
     "openssl": {

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4641,8 +4641,8 @@
       "port-version": 0
     },
     "libgwenhywfar": {
-      "baseline": "5.6.0",
-      "port-version": 5
+      "baseline": "5.12.0",
+      "port-version": 0
     },
     "libgxps": {
       "baseline": "0.3.2",

--- a/versions/l-/libgwenhywfar.json
+++ b/versions/l-/libgwenhywfar.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "5e874c9b061d1b706c0f3dcabd3bb96eeba9014e",
+      "git-tree": "2f617724273ebe192bfe9fa689a20936b51678fa",
       "version-semver": "5.12.0",
       "port-version": 0
     },

--- a/versions/l-/libgwenhywfar.json
+++ b/versions/l-/libgwenhywfar.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "797aa7b0ccc85eda3df568395239c8a68cfeaabb",
+      "git-tree": "967953014fe70315aae75089050989775b96613a",
       "version-semver": "5.12.0",
       "port-version": 0
     },

--- a/versions/l-/libgwenhywfar.json
+++ b/versions/l-/libgwenhywfar.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "8aa74036ba54602f978164a6dc85055adec577cd",
+      "git-tree": "35c7f16cf8e9c02e65d9ce898690ef2b8b0691dd",
       "version-semver": "5.12.0",
       "port-version": 0
     },

--- a/versions/l-/libgwenhywfar.json
+++ b/versions/l-/libgwenhywfar.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "967953014fe70315aae75089050989775b96613a",
+      "git-tree": "8aa74036ba54602f978164a6dc85055adec577cd",
       "version-semver": "5.12.0",
       "port-version": 0
     },

--- a/versions/l-/libgwenhywfar.json
+++ b/versions/l-/libgwenhywfar.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "5e874c9b061d1b706c0f3dcabd3bb96eeba9014e",
+      "version-semver": "5.12.0",
+      "port-version": 0
+    },
+    {
       "git-tree": "8b4376fdc006cc932f5bc581b32479ebd273b275",
       "version-semver": "5.6.0",
       "port-version": 5

--- a/versions/l-/libgwenhywfar.json
+++ b/versions/l-/libgwenhywfar.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "2f617724273ebe192bfe9fa689a20936b51678fa",
+      "git-tree": "797aa7b0ccc85eda3df568395239c8a68cfeaabb",
       "version-semver": "5.12.0",
       "port-version": 0
     },

